spec/integration/transport/winrm_spec.rb

```diff
@@ -1032,7 +1032,7 @@ def stub_winrm_to_raise(klass, message)
 
           with_tempfile_containing('script-test-winrm', contents, '.ps1') do |file|
             result = winrm.run_script(target, file.path, [])
-            expect(result).to be_success
+            expect(result).to_not be_success
```

Comment:
Could we also add a test that this now behaves as expected?

Reply:
`not_to`*

Reply:
Still interested in adding a test for the correct behavior

Reply:
Apologies for the delay in response, other work got in the way.

In reviewing the code again and this test, I added more detail in a comment in the affected test to explain to future people who read the code.

As requested, I added another test that ensures it returns 0 and succeeds.

Reply:
I'm working with the IAC team who will validate my changes and ensure they are working